### PR TITLE
CU/7055-Dylan-LoginWebviewScreens

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
Login and Webview screens

## Screenshots/Video
N/A

## Testing
yarn test

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
S src/App.test.tsx
S src/screens/BiometricsPreferenceScreen/BiometricsPreferenceScreen.test.tsx
S src/screens/auth/LoaGate/LoaGate.test.tsx
S src/screens/auth/LoginScreen/LoginScreen.test.tsx
S src/screens/WebviewScreen/WebviewControlButton.test.tsx
S src/screens/WebviewScreen/WebviewControls.test.tsx
S src/screens/WebviewScreen/WebviewTitle.test.tsx
S src/screens/WebviewScreen/WebviewScreen.test.tsx

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

**Review by theodur — CHANGES_REQUESTED** [review]
Seeing the same issues in this PR also
https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7148#pullrequestreview-1706588458

**Comment by Sparowhawk** [comment]
> Seeing the same issues in this PR also [#7148 (review)](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7148#pullrequestreview-1706588458)

updated

File: VAMobile/src/screens/auth/LoaGate/LoaGate.test.tsx [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
-        auth: { ...initialAuthState },
-      },
-    })
+    render(<LoaGate />)
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
+  it('initializes correctly', () => {
+    expect(screen.getByText('Sign in')).toBeTruthy()
```
Comment (theodur) [inline VAMobile/src/screens/auth/LoaGate/LoaGate.test.tsx:13]:
```suggestion
    expect(screen.getByRole('header', { name: 'Sign in' })).toBeTruthy()
```

File: VAMobile/src/screens/auth/LoaGate/LoaGate.test.tsx [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
+    expect(screen.getByText("Before we give you access to your VA claim and health care information, we need to make sure you’re you. This helps us protect you from fraud and identity theft.")).toBeTruthy()
+    expect(screen.getByText("If you haven’t yet verified your identity, we’ll help you complete the process when you sign in.")).toBeTruthy()
+    expect(screen.getByText("Read more if you haven't yet verified")).toBeTruthy()
```
Comment (theodur) [inline VAMobile/src/screens/auth/LoaGate/LoaGate.test.tsx:16]:
```suggestion
    expect(screen.getByRole('tab', { name: "Read more if you haven't yet verified" })).toBeTruthy()
```

**Review by theodur — CHANGES_REQUESTED** [review]

File: VAMobile/src/screens/auth/LoaGate/LoaGate.test.tsx (hunk 1 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/auth/LoaGate/LoaGate.test.tsx:13]: updated

**Review by alexandec — APPROVED** [review]

**Pull request merged**